### PR TITLE
task/01-setup

### PR DESCRIPTION
**Pull request by volodymyr-kravchenko-moc**
**Acceptance criteria for 1**

- [x] Mentor is added to pr
- [x] AC copied to pr description
- [x] link to deployed version added to pr
- [x] About block (text and image) added to the main page
- [x] On the widths 390px and 1280px, the site layout corresponds to the design

**Acceptance criteria for 1***

- [x] Created separate component About
- [x] Its styles added to .module.css
- [x] For App.tsx styles moved to App.module.css

https://volodymyr-kravchenko-moc.github.io/fe-react-2024/

### CODE REVIEW

**Comment by vladyslav-yermolin-moc** [comment]
Hey, @volodymyr-kravchenko-moc, thanks for your work.
I will check it during the day

File: src/components/about/About.module.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.wrapper {
+  display: flex;
+  column-gap: 100px;
+  padding: 50px;
+  height: 100%;
+}
+
+.imageMainWrap {
+  flex: 1 0 auto;
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.module.css]: I suggest you use full properties names, not the short description:
```css
flex-grow: 1;
flex-shrink 0;
flex-basis: auto;
```

Comment (vladyslav-yermolin-moc) [reply src/components/about/About.module.css]: Another solution: use `width` of the image, to set image's width, not the flex props of its container.

File: src/components/about/About.module.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  align-self: center;
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.module.css]:
```suggestion

```

File: src/components/about/About.component.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { AboutDescriptionComponent } from '@/components/about/description/AboutDescription.component.tsx';
+import { ImageComponent } from '@/components/common/image/Image.component.tsx';
+
+import styles from './About.module.css';
+
+export const AboutComponent = () => {
+    const imgAlt = 'About logo';
+
+    return (
+        <section className={styles.wrapper}>
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx:11]: I think, would be better to add description and image to one flex container and align its items by top side. Now you have unwanted indentation
image

File: src/components/about/About.component.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            <AboutDescriptionComponent />
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx]: Do you really need a separate component for this text? Will you plan to reuse it, or, maybe it has some logic, which you like to encapsulate?
This is just a text🤷‍♂️

File: src/components/about/description/AboutDescription.component.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { TitleComponent } from '@/components/common/title/Title.component.tsx';
+
+import styles from './AboutDescription.module.css';
+
+export const AboutDescriptionComponent = () => (
+    <div className={styles.box}>
+        <TitleComponent titleClasses={styles.aboutDescriptionTitle}>About me</TitleComponent>
+        <p>
+            Hi! My name is [Name] and I&apos;m a Junior Frontend Developer. I am already familiar with main Web Technologies like React,
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/description/AboutDescription.component.tsx]: Please add some name. We plan, hre you can add your name and photo. If you don't like this idea, just add some one.

File: src/components/about/description/AboutDescription.component.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+            HTML, CSS, JavaScript and Git version control system.
+        </p>
+        <p>This page was developed during the course &apos;Intro to React&apos; from Masters Academy in 2024.</p>
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/description/AboutDescription.component.tsx]: Please add a link as on the design
image

File: src/components/about/description/AboutDescription.component.tsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+        <p>
+            This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small
+            project for the portfolio.
+        </p>
+        <p>You can contact me via [social network name] and check out my GitHub.</p>
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/description/AboutDescription.component.tsx]: Please add some links. Your social networks or any other links.
image
Example from your colleague 
image

File: src/components/about/About.module.css [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.imageWrap {
+  max-width: 500px;
+}
+
+.img {
+  width: 100%;
+  height: auto;
+  border-radius: 20px;
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.module.css]: please correct the radius according to design: 30 for mobile, 40 for desktop

File: Dockerfile [hunk 9]
```diff
@@ -0,0 +1,3 @@
+FROM node:20.9.0
```
Comment (vladyslav-yermolin-moc) [inline Dockerfile:1]: cool😎

File: src/components/about/description/AboutDescription.module.css [hunk 10]
```diff
@@ -0,0 +1,18 @@
+.box {
+  flex: 1 1 auto;
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/description/AboutDescription.module.css]: This code does nothing, Would be better not to add it.

File: src/components/about/description/AboutDescription.module.css [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+  height: 100vh;
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/description/AboutDescription.module.css]: You have used 100vh for the main container already. 
here I suggest to dance from the parent size
```suggestion
  height: 100%;
```

File: src/components/about/About.component.tsx [hunk 12]
```diff
@@ -0,0 +1,18 @@
+import about from '@/assets/about/index.png';
```
Comment (vladyslav-yermolin-moc) [inline src/components/about/About.component.tsx]: Please add some decent names. Like
```suggestion
import aboutImg from '@/assets/about/about-photo.png';
```

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]
Good job 👍

File: src/components/about/description/AboutDescription.module.css (hunk 10 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/description/AboutDescription.module.css]: Fixed

File: src/components/about/About.component.tsx (hunk 12 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/About.component.tsx]: Fixed

File: src/components/about/About.module.css (hunk 8 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/About.module.css]: Fixed

File: src/components/about/About.component.tsx (hunk 4 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/About.component.tsx]: No, I just used it as a way to group elements
I realized it's not okay, so I removed the component :)

File: src/components/about/description/AboutDescription.module.css (hunk 11 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/description/AboutDescription.module.css]: Fixed

File: Dockerfile (hunk 9 above)
Comment (volodymyr-kravchenko-moc) [reply Dockerfile:1]: Thanks :)

File: src/components/about/description/AboutDescription.component.tsx (hunk 7 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/description/AboutDescription.component.tsx]: Fixed

File: src/components/about/description/AboutDescription.component.tsx (hunk 6 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/description/AboutDescription.component.tsx]: Fixed

File: src/components/about/description/AboutDescription.component.tsx (hunk 5 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/description/AboutDescription.component.tsx]: Fixed

File: src/components/about/About.component.tsx (hunk 3 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/About.component.tsx:11]: Fixed

File: src/components/about/About.module.css (hunk 1 above)
Comment (volodymyr-kravchenko-moc) [reply src/components/about/About.module.css]: Outdated

**Review by vladyslav-yermolin-moc — APPROVED** [review]
Nice start 👍

**Pull request merged**